### PR TITLE
Display custom html for property in data grid SN 519

### PR DESCRIPTION
**Pull request by toanphamsaritasa**
https://saritasa.atlassian.net/browse/SN-519

### CODE REVIEW

**Comment by toanphamsaritasa** [comment]
image

**Comment by vasiliy-chefonov** [comment]
@toanphamsaritasa Actually, we shouldn't specify any HTML templates. The only thing we need is to allow escaping HTML tags that can be already stored in DB, or can be returned by calculated fields. Let me know if you have any questions.

File: src/Saritasa.NetForge.Tests/PropertyCustomHtmlTests.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+    /// <summary>
+    /// Configure entity property through attribute configuration test.
+    /// </summary>
+    [Fact]
+    public void ConfigureProperty_WithCustomHtml_ThroughAttribute_Success()
+    {
+        // Arrange
+        var propertyInfo = typeof(Address).GetProperty(nameof(Address.Id));
+        var attribute = propertyInfo?.GetCustomAttributes(typeof(NetForgePropertyAttribute), false)
```
Comment (AlexanderMakarov124) [inline src/Saritasa.NetForge.Tests/PropertyCustomHtmlTests.cs:44]: Could we apply attribute inside this test? I think it isn't correct when test depends on manually applied attribute
If we by accidently remove this attribute test won't pass

**Review by AlexanderMakarov124 — APPROVED** [review]

File: src/Saritasa.NetForge.Tests/PropertyCustomHtmlTests.cs (hunk 1 above)
Comment (toanphamsaritasa) [reply src/Saritasa.NetForge.Tests/PropertyCustomHtmlTests.cs:44]: I believe that applying attribute operations will occur at design time to provide metadata to the class, whereas testing takes place at runtime. Therefore, modifying the attribute at runtime is not a good idea. Perhaps testing the configuration of the entity attribute is also not a good idea. Maybe I will remove the test then.

File: README.md [hunk 2]
```diff
@@ -451,3 +451,24 @@ optionsBuilder.ConfigureEntity<User>(entityOptionsBuilder =>
 [NetForgeProperty(IsExcludeFromQuery = true)]
 public string Property { get; set; }
 ```
+
+## Formatting property as HTML
+
+You can configure certain entity properties to rendered as custom HTML content in the data grid.
```
Comment (vasiliy-chefonov) [inline README.md:457]:
```suggestion
You can configure certain entity properties to be rendered as HTML content in the data grid.
```

File: src/Saritasa.NetForge.Tests/PropertyCustomHtmlTests.cs [hunk 3]
```diff
@@ -0,0 +1,34 @@
+﻿using Saritasa.NetForge.DomainServices;
+using Saritasa.NetForge.Tests.Domain.Models;
+using Xunit;
+
+namespace Saritasa.NetForge.Tests;
+
+/// <summary>
+/// Custom HTML for entity properties tests.
+/// </summary>
+public class PropertyCustomHtmlTests
```
Comment (vasiliy-chefonov) [inline src/Saritasa.NetForge.Tests/PropertyCustomHtmlTests.cs:10]: IMHO `PropertyFormatHtmlTests` sounds better.

**Review by vasiliy-chefonov — APPROVED** [review]

**Pull request merged**